app/models/comment.rb

```diff
@@ -1,2 +1,10 @@
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_commit :update_comment_counter, on: :create
+
+  def update_comment_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] OPTIONAL
It's a good practice to mark the `update_comments_counter` method as private, when `after_commit` method is implemented